packages/loader/container-loader/src/connectionManager.ts

```diff
@@ -500,6 +508,7 @@ export class ConnectionManager implements IConnectionManager {
                 if (retryDelayFromError !== undefined) {
                     this.props.reconnectionDelayHandler(retryDelayFromError, origError);
                 }
+                progress?.onRetry?.(delayMs, origError);
```

Comment:
This only works for IProgress from first call to connect(). If there are two calls in parallel, second caller does not get any feedback.